docs/book/index.md

```diff
@@ -1,15 +1,17 @@
-## Installation
+# Installation
 
-### Using Composer
+## Using Composer
```

Comment:
A blanket change at this point is unfortunately wrong, because then [the hierarchy for the page](https://docs.laminas.dev/laminas-cache/) is no longer correct.

Use `<!-- markdownlint-disable-next-line heading-increment -->`

See: https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md001.md

Reply:
Fixed as per your recommendation